Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -618,10 +681,55 @@ public function insertAt($index, $items)
         }
 
         for ($i = 0; $i < $itemsCnt; ++$i) {
+            if ('' === $items[$i]->getContent()) {
+                throw new \InvalidArgumentException('Must not add empty item to collection');
+            }
+
             $this[$i + $index] = $items[$i];
         }
     }
 
+    public function overrideRange($indexStart, $indexEnd, /** array|Tokens */ $items)
+    {
+        // TODO: add documentation
+        // 3 do wstawienia, 3 indexy                nadpisujemy
+        // 3 do wstawienia, 5 indexow               3 nadpisujemy, 2 kasujemy
+        // 3 do wstawienia, 1 nidex                 1 nadpisujemy, 2 dodajemy
```

Comment:
yea, it is already a TODO for me, see that first part: `// TODO: add documentation`